command/core.py

```diff
@@ -5,23 +5,26 @@
 class Core:
     """Encapsulate methods for handling events."""
 
-    def __init__(self):
+    def __init__(self, DBfacade, bot):
         """Initialize the dictionary of command handlers."""
         self.__commands = {}
-        self.__commands["user"] = UserCommand()
+        self._facade = DBfacade
+        self.__bot = bot
+        self.__commands["user"] = UserCommand(self._facade, self.__bot)
```

Comment:
This is fine for now but I think we’ll want a more principled way to do this in the future (e.g. for once we add the Github integration - that’ll need to be dependency injected too); I have some thoughts in that direction we can talk about this weekend.